docs/user/interop.rst

```diff
@@ -29,70 +29,87 @@ For example to call C's ``malloc`` one might declare it as following:
-ptrdiff_t             native.CPtrDiff (2)
-wchar_t               native.CWideChar
-char16_t              native.CChar16
-char32_t              native.CChar32
-float                 native.CFloat
-double                native.CDouble
-void*                 native.Ptr[Byte] (2)
-int*                  native.Ptr[native.CInt] (2)
-char*                 native.CString (2) (3)
-int (\*)(int)         native.CFunctionPtr1[native.CInt, native.CInt] (2) (4)
-struct { int x, y; }* native.Ptr[native.CStruct2[native.CInt, native.CInt]] (2) (5)
-struct { int x, y; }  Not supported
-===================== =========================
-
-(1) See `Unsigned integer types`_.
-(2) See `Pointer types`_.
-(3) See `Byte strings`_.
-(4) See `Function pointers`_.
-(5) See `Memory layout types`_.
+========================= =========================
+C Type                    Scala Type
+========================= =========================
+``void``                  ``Unit``
+``bool``                  ``native.CBool``
+``char``, ``signed char`` ``native.CChar``
+``unsigned char``         ``native.CUnsignedChar`` [1_]
+``short``                 ``native.CShort``
+``unsigned short``        ``native.CUnsignedShort`` [1_]
+``int``                   ``native.CInt``
+``unsigned int``          ``native.CUnsignedInt`` [1_]
+``long``                  ``native.CLong``
+``unsigned long``         ``native.CUnsignedLong`` [1_]
+``long long``             ``native.CLongLong``
+``unsigned long long``    ``native.CUnsignedLongLong`` [1_]
+``size_t``                ``native.CSize``
+``ptrdiff_t``             ``native.CPtrDiff`` [2_]
+``wchar_t``               ``native.CWideChar``
+``char16_t``              ``native.CChar16``
+``char32_t``              ``native.CChar32``
+``float``                 ``native.CFloat``
+``double``                ``native.CDouble``
+``void*``                 ``native.Ptr[Byte]`` [2_]
+``int*``                  ``native.Ptr[native.CInt]`` [2_]
+``char*``                 ``native.CString`` [2_] [3_]
+``int (*)(int)``          ``native.CFunctionPtr1[native.CInt, native.CInt]`` [2_] [4_]
+``struct { int x, y; }*`` ``native.Ptr[native.CStruct2[native.CInt, native.CInt]]`` [2_] [5_]
+``struct { int x, y; }``  Not supported
+========================= =========================
+
+.. [1] See `Unsigned integer types`_.
+.. [2] See `Pointer types`_.
+.. [3] See `Byte strings`_.
+.. [4] See `Function pointers`_.
+.. [5] See `Memory layout types`_.
 
 Linking with native libraries
 `````````````````````````````
 
-In C/C++ one has to typically pass an additional ``-l mylib`` flag to
-dynamically link with a library. In Scala Native one can annotate libraries
-to link with using ``@native.link`` annotation:
+C compilers typically require to pass an additional ``-l mylib`` flag to
+dynamically link with a library. In Scala Native, one can annotate libraries to
+link with using the ``@native.link`` annotation. As in C, it takes the library
+name without the ``lib`` prefix.
```

Comment:
> As in C, it takes the library name without the ``lib`` prefix.

I'm not sure this sentence makes things much clearer. I got a bit confused because `lib` is a suffix in the example above, plus this is not always true (take for instance zlib, you need to pass `-l z` to the linker). Maybe just say that you have to put whatever you would give your C compiler? I'm not a native English speaker, so that's really just a suggestion.